contracts/lib/Organization.sol

```diff
@@ -260,4 +314,39 @@ contract Organization is OrganizationInterface, IsMemberInterface {
         isWorker_ = workers[_worker] > block.number;
     }
 
+
+    /* Private functions */
+
+    /**
+     * @notice Sets worker and its expiration block height.
+     *
+     * @param _worker Worker address to be added.
+     * @param _expirationHeight Expiration block height of worker.
+     *
+     * @return remainingBlocks_ Remaining number of blocks for which worker is
+     *                          active.
+     */
+    function setWorkerInternal(
```

Comment:
What about combining setWorker and setWorker internal functions as a one and make it public?

Reply:
How would that work with the `external` interface?

Reply:
That's why I suggest to make it `public`. It is an additional call we do each time.
However, I am open to listen why do you prefer it in this way.

Reply:
Can it be declared `external` in the  interface and be implemented `public` in the contract?

Reply:
I am going to say yes (needs to be checked), as it's widening the accessibility scope of the function. Public methods can be called internally and externally, so we do not break the interface design of the interface contract.

Reply:
It would indeed work! However, there is another problem: the external function should be restricted to `onlyOwnerOrAdmin` while the internal call should not (it is called on construction). Ideas?